.azure-pipelines.yml

```diff
@@ -269,6 +269,34 @@ jobs:
 #################################################################################
 
 
+- job: 'linux_pypi_build'
+  displayName: 'Linux PyPI build'
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+  - checkout: self
+    submodules: true
+  - bash: |
+      set -eux
+      sudo apt install python python-pip
+      pip install twine
+    displayName: 'Bootstrap'
+  - bash: |
+      set -eux
+      python setup.py generate_pyqt sdist
+  - bash: |
+      echo $SNAPCRAFT_CREDENTIALS | twine upload dist/*
```

Comment:
Wouldn't it be better to use the create a `~/.pypirc` instead ? (see https://packaging.python.org/specifications/pypirc/#using-a-pypi-token)

Reply:
So we would include the pypirc in our private vars and echo it to the file?

Reply:
I think it would be less cumbersome to do:
```shell
echo "
[pypi]
username = __token__
password = $SNAPCRAFT_CREDENTIALS
" > ~/.pypirc
```

Reply:
Done